content/posts/my_first_post.md

```diff
@@ -6,7 +6,7 @@ tags: [short story]
 slug: my_first_post
 ---
 
-We begin as the setting sun purples the sky. There's a few pink clouds hanging overhead, glowing slightly as the last few rays of sun filter through them. The sky is a thousand-acre watercolor, an endless yet paper-thin canvas that only nature, or perhaps the supernatural, has the nimbleness paint on, a masterpiece that so many artists have tried and failed to tame for themselves.
+We begin as the setting sun purples the sky. A few pink clouds hang overhead, glowing slightly as the last few rays of sun filter through. The sky is a thousand-acre watercolor, an endless yet paper-thin canvas that only nature, or perhaps the supernatural, has the nimbleness to paint on, a masterpiece that so many artists have tried and failed to tame for themselves.
 
 I kick off and start pedaling in the street, with him not too far behind. Soon, the sodium-vapor streetlamps flicker on and tint the streets with their orange light. We head north leisurely, cruising through the streets with the ghosts of cars following us. After a while, we stop near a brook on the edge of town. Dusk falls and night inches closer, slowly bringing layers of deep blues and dark hues into the sky. The water trembles slightly as the wind ripples its surface.
```

Comment:
I don't love 
> we head north leisurely
I think it could use a better verb? like "meander" or "snake" or something else that summarizes "head + leisure"?

I like the "ghosts of cars following us" a lot. If you wanted to be more explicit, you could say "in the corners of our eyes" or something about the beams?

Don't love the juxtaposition of "Dusk falls and night inches closer" - they don't feel like complementary movements. The last sentence is good, feels pretty evocative.